site/src/components/Breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import {
+	Breadcrumb,
+	BreadcrumbEllipsis,
+	BreadcrumbItem,
+	BreadcrumbLink,
+	BreadcrumbList,
+	BreadcrumbPage,
+	BreadcrumbSeparator,
+} from "components/Breadcrumb/Breadcrumb";
+import { MockOrganization } from "testHelpers/entities";
+
+const meta: Meta<typeof Breadcrumb> = {
+	title: "components/Breadcrumb",
+	component: Breadcrumb,
+};
+
+export default meta;
+type Story = StoryObj<typeof Breadcrumb>;
+
+export const Default: Story = {
+	args: {
+		children: (
+			<Breadcrumb>
+				<BreadcrumbList>
+					<BreadcrumbItem>
+						<BreadcrumbPage>Admin Settings</BreadcrumbPage>
+					</BreadcrumbItem>
+					<BreadcrumbSeparator />
```

Comment:
I feel like inserting the separator shouldn't be the responsibility of the consumer. The `BreadcrumbList` could use `Children.toArray` and intersperse them automatically, you could use some CSS tricks with `:not(:first-of-kind)` or whatever, etc. There are other options that reduce the risk of misusage.

Also why are `<Breadcrumb>` and `<BreadcrumbList>` even separate components?

Reply:
I was originally thinking that the Breadcrumb should be more universal so that it could be used in places without the separator line underneath and this is how shadcn/ui does it as well. Of course, if the argument is that breadcrumbs for Coder will always have a separator line underneath, it could make sense.

Reply:
Having both <Breadcrumb> and <BreadcrumbList> is a common pattern in the shadcn/ui components to add some semantic meaning. In general, I am not focused on altering or adding opinions to the way the components are constructed but always open to suggestions.